hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouterNetworkTopologyServlet.java

```diff
@@ -204,7 +279,41 @@ public void testPrintTopologyNoDatanodesJsonFormat() throws Exception {
     sb.append("\n-- Network Topology -- ");
     String topology = sb.toString();
 
-    // assert node number
+    // Assert node number.
     assertTrue(topology.contains("No DataNodes"));
   }
 }
+
+class RouterServerHelper implements BeforeEachCallback, AfterAllCallback {
```

Comment:
@hfutatzhanghb Thank you for your contribution! I will review this PR today and share some thoughts later.